packages/common/src/users/HubUsers.ts

```diff
@@ -55,16 +55,8 @@ export async function enrichUserSearchResult(
     result.isGroupOwner = user.isGroupOwner;
   }
 
-  // Informal Enrichments - basically adding type-specific props
-  // derived directly from the entity
-
-  // default includes
-  const DEFAULTS: string[] = ["org.name AS orgName"];
-
-  // merge includes
-  include = [...DEFAULTS, ...include].filter(unique);
   // Parse the includes into a valid set of enrichments
-  const specs = include.map(parseInclude);
+  const specs = include.filter(unique).map(parseInclude);
```

Comment:
No longer force the portal enrichment for all user searches, the `user` & `groupMember` `targetEntity`ies still gets that enrichment though by explicitly passing it now to avoid breaking change